lib/MetaCPAN/Server/Controller/Favorite.pm

```diff
@@ -45,5 +45,12 @@ sub recent : Path('recent') : Args(0) {
     $c->stash($data);
 }
 
+sub leaderboard : Path('leaderboard') : Args(0) {
+    my ( $self, $c ) = @_;
+    my $data = $self->model($c)->raw->leaderboard();
+    $data or return;
+    $c->stash($data);
```

Comment:
you can't stash a non-hashref value ... that's where this check comes from.
i don't mind the suggested change, but i assigned $data for debugging purposes.